subt_ign/include/subt_ign/CommonTypes.hh

```diff
@@ -66,8 +66,8 @@ namespace subt
   enum class ArtifactType : uint32_t
   {
     TYPE_BACKPACK = 0,
-    TYPE_DUCT,
     TYPE_DRILL,
+    TYPE_DUCT,
```

Comment:
Was this change intentional?

Reply:
Yes, so that the indices match this: https://github.com/osrf/subt/blob/master/subt_ign/src/GameLogicPlugin.cc#L78